mcux/mcux-sdk/devices/MIMX8QM6/drivers/fsl_clock.c

```diff
@@ -375,3 +415,138 @@ void CLOCK_ConfigLPCG(clock_ip_name_t name, bool swGate, bool hwGate)
             break;
     }
 }
+
+void CLOCK_ConfigLPCGMapped(uint32_t *lpcgBase, clock_ip_name_t name, bool swGate, bool hwGate)
```

Comment:
Is it possible to consider code reusing for the new added *Mapped functions, rather than duplication?
For example,
CLOCK_ConfigLPCG could just invoke CLOCK_ConfigLPCGMapped with lpcgBase LPCG_TUPLE_REG_BASE(name) provided?

Thanks.

Reply:
Done.

Reply:
The changes look good to me.
Thanks for the quick response.